From c5fa0b778e1c2a7d03ff6e661bdfa2faef878f68 Mon Sep 17 00:00:00 2001
From: Romain Naour <romain.naour@smile.fr>
Date: Fri, 1 Jul 2016 17:56:30 +0200
Subject: [PATCH] configure: remove po/Makefile.in

The file po/Makefile.in is automatically added to AC_OUTPUT while using gettexize

Signed-off-by: Romain Naour <romain.naour@smile.fr>
---
 configure.ac | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index f7fb40f..a25e463 100644
--- a/configure.ac
+++ b/configure.ac
@@ -55,5 +55,4 @@ dnl last but not least
 AC_OUTPUT([Makefile
 	libuio.dox
 	libuio-uninstalled.pc
-	libuio.pc
-	po/Makefile.in])
+	libuio.pc])
-- 
2.5.5

